sources/vega/vega.proto

```diff
@@ -938,6 +938,24 @@ message NetworkParameter {
   string value = 2;
 }
 
+// Network limits, defined in the genesis file
+message NetworkLimits {
+  // Are market proposals allowed at this point in time
+	bool can_propose_market = 1;
+  // Are asset proposals allowed at this point in time
+  bool can_propose_asset = 2;
+  // True once block count > bootstrapBlockCount
+  bool bootstrap_finished = 3;
+  // Are market proposals enabled on this chain
+	bool propose_market_enabled = 4;
+  // Are asset proposals enabled on this chain
+  bool propose_asset_enabled = 5;
```

Comment:
Should we add the *_enabled_from fields which are times?

Reply:
I had them there originally but they were coming back as zero times, which apparently don't have a defined representation as an integer (https://pkg.go.dev/time#Time.UnixNano). Should they be something non-zero, or I guess we could use a string representation or something instead?